### DISCUSSION TITLE
Download to a S3 bucket

### DISCUSSION THREAD

**Discussion by remi-braun** [CS-SI/eodag#904] [General]
Hello,

I think it's not , but is it possible to download directly a product to a S3 (-compatible) bucket ?
I think the downloading path management is not cloud-safe, but I am maybe missing something.

I tried with a link (`https://s3.unistra.fr/test/`) and a S3-path (`s3://test/`) with our credentials.

**Comment by sbrunato** [comment]
Hello @remi-braun , no this is not handled for the moment

**Reply by remi-braun** [reply]
OK no problem !

**Comment by sbrunato** [comment]
A tuto is available for that using `stream_download` method (method introduced in `v4.1.0`) 
https://eodag.readthedocs.io/en/stable/notebooks/tutos/tuto_stream_download_to_s3.html
